.ci/legacy_project/README.md

```diff
@@ -0,0 +1,26 @@
+This directory contains a partial snapshot of an old Flutter project; it is
+intended to replace the corresponding parts of a newly Flutter-created project
+to allow testing plugin builds with a legacy project.
+
+It was originally created with Flutter 2.0.6. In general the guidelines are:
+- Pieces here should be self-contained; avoid making a hybrid of old and new
+  projects except at major cut points (e.g., platform directories). This is
```

Comment:
Can you expound on this a bit in the review? What is a major cut point? The example of platform directories didn't help me understand how to evaluate if flutter has hit another major cut point or not.

Reply:
I rewrote it to hopefully be clearer. What I was describing as "cut points" was meant to be about project structure, not Flutter releases. So for instance "the android directory" is a major cut point, whereas "the Java code from the android directory" would not be, because it can be strongly coupled to resources or configs that are not within that subset.